packages/store-sync/src/postgres/columnTypes.ts

```diff
@@ -73,6 +57,20 @@ export const asAddress = (name: string) =>
     },
   })(name);
 
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+export const asBoolArray = (name: string) =>
+  customType<{ data: boolean[]; driverData: string[] }>({
+    dataType() {
+      return "bool[]";
+    },
+    toDriver(data: boolean[]): string[] {
+      return data.map((datum) => String(datum));
+    },
+    fromDriver(driverData: string[]): boolean[] {
+      return driverData.map((datum) => Boolean(datum));
+    },
```

Comment:
@holic do you have any intuition why these drivers don't work?

As `Uint8Array` is interpreted as `bytea`, I would expect `Uint8Array[]` to be interpreted as `bytea[]`. Yet this errors with `PostgresError: column "value" is of type bytea[] but expression is of type bytea`. Test run:

https://github.com/latticexyz/mud/actions/runs/7889740453/job/21530302035?pr=2251#step:6:1813

Reply:
I'm not sure! I do recall seeing some weird behavior with drizzle driver and the way it translates between types.

Reply:
fyi to future readers: I meant to highlight the bytes drivers, not boolean.